text/src/autogluon/text/automm/optimization/metrics.py

```diff
@@ -0,0 +1,54 @@
+import torch
+from torch.nn import functional as F
+from torchmetrics import Metric
+
+
+class CrossEntropy(Metric):
```

Comment:
We will need to add a test-case for CrossEntropy. May borrow some example codes from https://github.com/PyTorchLightning/metrics/blob/5cbc7add8877e6524b140eb35f57a26b53fe5670/tests/regression/test_mean_error.py#L88-L140

Reply:
Test cases added.